lib/rules/selector-max-class/README.md

```diff
@@ -0,0 +1,51 @@
+# selector-max-compound-selectors
+
+Limit the number of classes in a [complex selector](https://www.w3.org/TR/selectors4/#complex).
+
+```css
+   div .foo.bar[data-val] > a.baz {}
+/*     ↑   ↑                 ↑
+       |   |                 |
+       1   2                 3  -- this selector contains three classes */
+```
+
+This rule resolves nested selectors before counting the number of classes in a selector. Each selector in a [selector list](https://www.w3.org/TR/selectors4/#selector-list) is evaluated separately.
+
+The `:not()` pseudo-class is also evaluated separately. The rule processes the argument as if it were an independent selector, and the result does not count toward the total for the entire selector.
```

Comment:
I can't argue with you, because my English is not good enough. Thought it was a typo.